includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1619,6 +1619,16 @@ public static function filter_tag_row_actions( $actions, WP_Term $tag ) {
 				esc_attr__( 'Toggle error details', 'amp' ),
 				esc_html__( 'Details', 'amp' )
 			);
+
+			$json           = json_decode( $term->description, true );
```

Comment:
So for a single error the JSON would look like:

```json
{
  "code": "DISALLOWED_TAG",
  "node_attributes": {
    "src": "https:\/\/wordpressdev.lndo.site\/content\/themes\/twentytwentyone\/assets\/js\/primary-navigation.js?ver=__normalized__",
    "id": "twenty-twenty-one-primary-navigation-script-js"
  },
  "node_name": "script",
  "node_type": 1,
  "parent_name": "body",
  "type": "js_error",
  "status": "Removed"
}
```

For the `node_type` field, I'm wondering if we should get the XML constant name from the value and use that instead. By itself, the constant value wouldn't be of much use and would require the person having to lookup the constants themselves. For reference, here's [the list](https://www.php.net/manual/en/dom.constants.php):

![image](https://user-images.githubusercontent.com/16200219/96514406-2490ff80-1253-11eb-8ff1-ee6d935f75c9.png)

Reply:
We could reflect the [`XMLReader`](https://www.php.net/manual/en/class.xmlreader.php) class and obtain the constants from there

Reply:
Done in fe358015b